### PR TITLE
Cisco Webex List Meeting (Independent Publisher)

### PR DESCRIPTION
**Pull request by last72**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

hostEmail blurred for security reason
image
image

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hello @last72,

Thank you so much for submitting this new proposal. We are looking forward to seeing your connector artifacts.

Please let us know if you have any questions.

Thank you so much again for your hard work with us.

**Comment by last72** [comment]
@Amjed-Ayoub Removed "Proposal" prefix.

File: independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json:9]: @last72 please add the email property to the contact section

File: independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json:24]: @last72 please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json:54]: @last72 I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: independent-publisher-connectors/Cisco Webex List Meeting/apiProperties.json
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Cisco Webex List Meeting/apiProperties.json:3]: @last72 in the swagger file you are mentioning OAuth type. can you please fix the apiProperties.json file as well?

File: independent-publisher-connectors/Cisco Webex List Meeting/readme.md
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Cisco Webex List Meeting/readme.md:1]: @last72 please include all required sections in the readme file. please see our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
> @Amjed-Ayoub This seems to be subset of #2286 Can you merge this PR to that PR please?

Hello @last72,

Thank you so much for bringing this to my attention.

Can you please clarify on this? is this the same connector?
If yes, then please wait until we merge the [other PR](https://github.com/microsoft/PowerPlatformConnectors/pull/2286). and then you can submit and update to the same connector.
Please feel free to close this PR.

Please let me know if you have any questions.

**Comment by last72** [comment]
@Amjed-Ayoub Yes. It seems to be pointing to same end point with same action. I missed that the other connector was in progress.

**Comment by Amjed-Ayoub** [comment]
@last72 can you please confirm if we should close this PR?

**Comment by vmanoharas** [comment]
Hello @last72 

We are closing this PR as per the above conversation, please submit new PR.

**Pull request closed without merging**
